scrapy/downloadermiddlewares/httpcache.py

```diff
@@ -70,7 +70,7 @@ def process_request(self, request: Request, spider: Spider) -> Optional[Response
             self.stats.inc_value('httpcache/miss', spider=spider)
             if self.ignore_missing:
                 self.stats.inc_value('httpcache/ignore', spider=spider)
-                raise IgnoreRequest("Ignored request not in cache: %s" % request)
+                raise IgnoreRequest(f"Ignored request not in cache: {str(request)}")
```

Comment:
```suggestion
                raise IgnoreRequest(f"Ignored request not in cache: {request}")
```
Use `str` inside a f-string is redudant, same for others places.

Reply:
Ok, thanks @Laerte

Reply:
@GeorgeA92 No problem, great work btw.

Reply:
I think this wasn't implemented?